### PR TITLE
Create polls index page

### PR DESCRIPTION
**Pull request by saeedmb8**

### CODE REVIEW

**Comment by saeedmb8** [comment]
@miltontony ready for review :)

File: molo/polls/migrations/0003_create_polls_index_pages.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    from molo.core.models import Main
+    from molo.polls.models import Question, FreeTextQuestion, PollsIndexPage
+    main = Main.objects.all().first()
+
+    if main:
+        polls_index = PollsIndexPage(title='Polls', slug='polls')
+        main.add_child(instance=polls_index)
+        polls_index.save_revision().publish()
+
+        # Move existing Questions
```
Comment (miltontony) [inline molo/polls/migrations/0003_create_polls_index_pages.py]: I think this migration should stop here, and the rest should be handled on an app level

File: molo/polls/templatetags/poll_votings.py [hunk 2]
```diff
@@ -3,16 +3,17 @@
...
 
-from molo.polls.models import Question, Choice
+from molo.polls.models import Question, Choice, PollsIndexPage
 
 register = template.Library()
 
 
 @register.inclusion_tag('polls/poll_page.html',
                         takes_context=True)
-def poll_page(context, pk=None, page=None):
```
Comment (miltontony) [inline molo/polls/templatetags/poll_votings.py]: This should all take a `page=None`

File: molo/polls/templatetags/poll_votings.py [hunk 3]
```diff
@@ -3,16 +3,17 @@
...
+def poll_page(context, pk=None):
     context = copy(context)
     locale_code = context.get('locale_code')
+    page = PollsIndexPage.objects.live().all().first()
```
Comment (miltontony) [inline molo/polls/templatetags/poll_votings.py]: And here it should say 
```python 
if page is None:
    page = PollsIndexPage.objects.live().first()
```

**Comment by saeedmb8** [comment]
@miltontony ready for review :)

**Comment by miltontony** [comment]
:+1:

**Pull request merged**